### DISCUSSION TITLE
course: Angular Material Components Theming System: Complete Guide | Modify typography | Angular Material Dev

### DISCUSSION THREAD

**Discussion by giscus** [Angular-Material-Dev/community#13] [Article and Course Discussions]
# course: Angular Material Components Theming System: Complete Guide | Modify typography | Angular Material Dev

Learn Material Design in Angular, Theming Angular Material Components and Developing your Angular application with Material Design like a pro.

https://angular-material.dev/courses/m2-ng-components/m2-ng-components/modify-typography

**Comment by akaustav** [comment]
Using Angular CLI 15.2.11, when I attempt to issue the following command:
```shell
ng new my-app --style=scss --defaults --skip-tests --standalone
```

I receive an error stating:
```shell
Error: Unknown argument: standalone
```

When I run the command without the standalone flag - like the following - it works:
```shell
ng new my-app --style=scss --defaults --skip-tests
```

**Reply by shhdharmen** [reply]
I would recommend to update your project to Angular v17. Because all the content is written w.r.t. v17. For update guide, you can refer to https://angular-material.dev/courses/m2-ng-components/m2-ng-components/update-guide

**Reply by akaustav** [reply]
**Side note:** There seems to be something wrong with the way these comments are being associated with the respective tutorial pages. I had originally put this particular thread in the comments section at the bottom of the [Setting up the project](https://angular-material.dev/courses/m2-ng-components/m2-ng-components/setting-up-the-project) page.

**Reply by akaustav** [reply]
@shhdharmen - Thanks for the response. We intend to upgrade our project to Angular 17 and beyond eventually. But we're going in steps from one major version of Angular to the next. And right now we're transitioning our components to comply with the MDC design guidelines. We are using your excellent guide to understand and navigate some poorly documented intricacies that are involved with this migration.

Since the text and the table at the top of the [Overview](https://angular-material.dev/courses/m2-ng-components/m2-ng-components/overview) page shows that this guide was compatible with Angular Material version 15, 16 and 17, I was merely pointing out the facts for anyone else following along your guide who might run into similar hurdles as we did.

**Reply by shhdharmen** [reply]
You are right, this guide should work as intended for Angular versions 15, 16 and 17 with respect to scss and Angular Material.

As far as module and standalone is concerned, content is written for standalone mode. Except that, you should be able to follow the guide for all 3 versions.
